src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+body {
+  margin: 0 10;
```

Comment:
The `margin` property value `0 10` is incorrect. It should specify units for the margin values, such as `0 10px` or `0 10%`.